### DISCUSSION TITLE
How can connect DAPP in page like Gnosis Safe?

### DISCUSSION THREAD

**Discussion by totom123** [WalletConnect/walletconnect-monorepo#4113] [Developer Support] [answered]
I am a junior developer
Anyone have idea how to connect DAPP without open model and copy the code?
Look like Gnosis Safe
[https://app.safe.global/apps](https://app.safe.global/apps)
Thanks

**Comment by glitch-txs** [comment, marked as answer]
Hi, you can check our web examples here: https://github.com/WalletConnect/web-examples

**Reply by totom123** [reply]
thanks for ur answer
I have tried every example but no one connect like Gnosis Safe
Do u have any suggestion?
thanks

**Reply by glitch-txs** [reply]
You have to use ethereum-provider and disable the modal, the Uri is what you copy https://docs.walletconnect.com/advanced/providers/ethereum

**Reply by totom123** [reply]
thanks for ur reply
Is it means I must get the DAPP project id first before I create the Uri?
thanks
